lib/solarwinds_apm/otel_native_config.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+# © 2023 SolarWinds Worldwide, LLC. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at:http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+
+require 'solarwinds_apm/constants'
+require 'solarwinds_apm/api'
+require 'solarwinds_apm/support'
+require 'solarwinds_apm/opentelemetry'
+require 'solarwinds_apm/sampling'
+
+module SolarWindsAPM
+  # OTelNativeConfig module
+  module OTelNativeConfig
```

Comment:
What are the differences between the new `OTelNativeConfig` and current `OTelConfig`?

How come `OTelConfig` is initialized in solarwinds_apm.rb but not `OTelNativeConfig`?

What are the conditions or use case for using one instead of the other? Or are they both used?

Reply:
OTelConfig is still used liboboe, OTelNativeConfig use pure ruby sampler.
I keep using OTelConfig to avoid test case fail on other non-ruby sampler related test case.
Naming for `OTelNativeConfig` is also temporary as I kind want to focus on the logic of pure ruby sampler

Reply:
Thanks! Sure, that works